backend/infrahub/core/constants/__init__.py

```diff
@@ -113,13 +113,18 @@ class RepositoryAdminStatus(InfrahubStringEnum):
     STAGING = "staging"
 
 
-class RepositoryStatus(InfrahubStringEnum):
+class RepositorySyncStatus(InfrahubStringEnum):
     UNKNOWN = "unknown"
-    SYNCING = "syncing"
-    INSYNC = "in-sync"
-    ERROR_CRED = "error-cred"
+    IN_SYNC = "in-sync"
```

Comment:
I think it would be good to keep `SYNCING = "syncing"` or something similar to indicate that an action is ongoing

Reply:
The way I was thinking was that a syncing or action indicator would lead to a constantly flapping state where a repo would typically transition from in-sync to syncing and back again every 10 seconds, alternatively from an error state to syncing and back again which would build up to a huge number of old relationships in the database.  What are your thoughts around this?

Reply:
I would expect the state to change only if we have a new commit ... which shouldn't happen that often

Reply:
Ah sorry, my bad I was thinking about the operational sync status. Yes this makes sense. I'll add that.

Reply:
I brought in the latest changes from the develop branch and also rebased this one with my update so it might be hard to spot the difference but the changes here are that I kept the SYNCING status so that it's available on the sync_status field and then within the .import_objects_from_files() method I've added this at the top:

```python
await self._update_sync_status(branch_name=infrahub_branch_name, status=RepositorySyncStatus.SYNCING)
```

So please take another look at this.